schemas/com.snowplowanalytics.snowplow/cross_navigation/jsonschema/1-0-0

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a user and session identifiers entity included in cross platform or cross domain navigation.",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow",
+        "name": "cross_navigation",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "user_id": {
+            "description": "The business user ID, the user was identified in the source app.",
+            "type": [
+                "null",
+                "string"
+            ],
+            "maxLength": 4096
+        },
+        "domain_user_id": {
+            "type": "string",
+            "maxLength": 128,
+            "description": "Domain user ID of the source app from which a link was followed. The corresponding atomic property is named refr_domain_userid."
+        },
+        "session_id": {
+            "type": [
+                "null",
+                "string"
+            ],
+            "format": "uuid",
+            "description": "Session ID of the source app from which a link was followed."
+        },
```

Comment:
Why do we use `domain_user_id` but `session_id`? Also in the atomic table it's `domain_userid` without the extra underscore?

Reply:
Also how is `session_id` used, we specify where `domain_userid` ends up but not this or user_id?

Reply:
It seemed unnecessary to add the `domain_` in front of the `session_id`, but we had to do that in order to distinguish between `user_id` and `domain_user_id`.

The `userid` in `domain_userid` spelling is weird and doesn't match with the other `user_id`. Not sure if we should stick to our past mistakes... Also a bit lazy to change it at this point since we already have an approved enrich PR with it 😅 

Sorry, I didn't get the second question, what do you mean by specifying the `domain_userid`? This information is attached by the tracker to the URL get params automatically (if configured) and then enrich takes it out of the params and populates the entity.